packages/core/src/lib/managers/keyboard.manager.ts

```diff
@@ -0,0 +1,37 @@
+import { BehaviorSubject } from 'rxjs';
+import { CopyAction } from '../actions/copy.action';
+import { DeleteKeyPressed } from '../actions/delete-key-pressed.action';
+import { PasteAction } from '../actions/paste.action';
+import { DiagramEngineCore } from '../engine.core';
+import { NodeModel, PointModel, PortModel, LinkModel } from '../models';
+
+export class KeyboardManager {
```

Comment:
we need a dispose function to destruct and remove all used resources and references.

Reply:
still missing the dispose function that cleans up resources